client/extensions/woocommerce/app/products/product-form-simple-card.js

```diff
@@ -87,14 +95,17 @@ const ProductFormSimpleCard = ( { siteId, product, editProduct, translate } ) =>
 		'products__product-form-stock-disabled': ! product.manage_stock,
 	} );
 
+	const { stock_quantity } = product;
+	const quantity = ! isNull( stock_quantity ) && Number( stock_quantity ) >= 0 ? stock_quantity : '';
```

Comment:
Is it possible to have a negative stock quantity? Back orders? Just curious what the check here is trying to prevent. Seems strange to be doing API data validation in a render function.